docs/pages/components/side-navigation.md

```diff
@@ -394,54 +401,54 @@ The following requirements must be met:
 </ul>
 {% capture default %}
 <nav class="fd-side-nav fd-side-nav--condensed">
+    <a class="fd-side-nav__skip-link" href="#content">Skip navigation</a>
     <div class="fd-side-nav__main-navigation">
         <ul class="fd-nested-list">
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--home"></span>
+                    <span aria-label="Level 1 Item" class="fd-nested-list__icon sap-icon--home"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--calendar"></span>
+                    <span aria-label="Level 1 Item" class="fd-nested-list__icon sap-icon--calendar"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
                 <div class="fd-nested-list__content is-selected has-child" tabindex="0">
```

Comment:
I believe the tabindex here can be removed as the button now receives focus (there is currently a duplicate tab stop).